addons/udes_priorities/models/priority.py

```diff
@@ -107,7 +121,11 @@ def _check_for_oustanding_pickings(self):
             if priority._number_of_oustanding_pickings() > 0:
                 priorities_with_outstanding_picks |= priority
 
-        if priorities_with_outstanding_picks:
+        privileged_user = self.env.uid == SUPERUSER_ID or \
+            self.env.user.has_group("udes_security.group_debug_user") or \
+            self.env.user.has_group('udes_security.group_trusted_user')
+
+        if priorities_with_outstanding_picks and not privileged_user:
```

Comment:
Is `priorities_with_outstanding_picks` the same as `affected_pickings`?

Reply:
I haven't given too much thought on this because it's simply avoiding users without access rights to modify priorities but the priorities edit form cannot be seen by anyone else apart from trusted users. Since we are now allowing trusted users to edit priorities completely we could avoid entirely checking whether there are outstanding priorities since we are showing these on the form view. We could simply check for user privileges as a validation step. Does that sound clear?

Reply:
It is fine as it is, we only have to consolidate the compute/usage of number of outstanding pickings / affected pickings.